api/SWR/CarePackage/useCarePackageApi.js

```diff
@@ -7,6 +7,8 @@ const useCarePackageApi = {
   summary: (carePackageId) =>
     useGetData(carePackageId !== undefined ? `${CARE_PACKAGES_URL}/${carePackageId}/summary` : null),
 
+  approvers: () => useGetData('/users/approver', 'Can not get approvers', []),
```

Comment:
Let's move this hook to `SWR/users.js` file

Also, let's create a new hook `useApprovers` instead of putting it in the API object. In such case, these are not hooks, but just plain functions, and I think they are not handled appropriately by React